package.json

```diff
@@ -19,5 +19,9 @@
   "bugs": {
     "url": "https://github.com/KonomiAI/Chronicle/issues"
   },
-  "homepage": "https://github.com/KonomiAI/Chronicle#readme"
+  "homepage": "https://github.com/KonomiAI/Chronicle#readme",
+  "dependencies": {
+    "@dropzone-ui/react": "^6.7.1",
+    "axios": "^0.26.0"
+  }
```

Comment:
I think you added the dependencies in the wrong package.json, they should be placed in `client/package.json`, not the root one.